src/Pages/login.page.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+class login extends React.Component {
+	render() {
```

Comment:
to follow up with `team conventions` this file deserves to go under `src/App/Components/login/index.js`

Reply:
this is not the actual component mn, it's just a test. When we'll work on the real login feature, it will be replaced

Reply:
I just wanted it become like a `guidance` to other teammates

Reply:
And by the way wy should we have a folder named login?

Reply:
if you are aware of the long run, a single component like login, signup.. can have its children components that will need to be wrapped together inside a single folder.

Reply:
Components should go under a folder named component. We should emphasize on making them reusable. The same components used for login can be used for signup for example...

Reply:
Okay, that makes sense